### PR TITLE
add skipped attribute to junit xml testsuite

### PR DESCRIPTION
**Pull request by brycekahle**

### CODE REVIEW

File: internal/junitxml/report.go [hunk 1]
```diff
@@ -33,6 +33,7 @@ type JUnitTestSuite struct {
 	XMLName    xml.Name        `xml:"testsuite"`
 	Tests      int             `xml:"tests,attr"`
 	Failures   int             `xml:"failures,attr"`
+	Skipped    int             `xml:"skipped,attr"`
```
Comment (ccoVeille) [inline internal/junitxml/report.go:36]:
```suggestion
	Skipped    int             `xml:"skipped,attr,omitempty"`
```


I would prefer to do not have the attribute when not needed

Comment (brycekahle) [reply internal/junitxml/report.go]: Done!

Comment (ccoVeille) [reply internal/junitxml/report.go]: Thanks

**Comment by brycekahle** [comment]
Anything I can do to help get this merged?

**Review by ccoVeille — APPROVED** [review]
I give you my vote. Unfortunately, I'm a random gopher

**Review by dnephin — APPROVED** [review]
Thank you for the PR! This is great

**Pull request merged**